src/main/java/mate/academy/util/HashUtil.java

```diff
@@ -0,0 +1,28 @@
+package mate.academy.util;
+
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+
+public class HashUtil {
+
+    private static final String HASH_ALGORITHM = "SHA-512";
+
```

Comment:
create private constructor for this Util class so it can't be instantiated mistakenly